orbita.php

```diff
@@ -219,9 +219,13 @@ function orbita_get_post_html( $post_id ) {
 		return;
 	}
 
+	$separator = '?';
+	if(strpos($external_url, '?') !== FALSE) {
```

Comment:
```suggestion
	if(strpos($external_url, '?') !== false) {
```

o que acha de manter minúsculo pra seguir o padrão do restante do arquivo?

Reply:
justo! 😉